questionnaires/ralpmh_covid_symptoms/ralpmh_covid_symptoms_armt.json

```diff
@@ -0,0 +1,648 @@
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "None "
+            },
+            {
+                "code": "1",
+                "label": "Mild "
+            },
+            {
+                "code": "2",
+                "label": "Moderate "
+            },
+            {
+                "code": "3",
+                "label": "Severe"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "y",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "covid_symptom_common",
+        "matrix_ranking": "",
+        "field_annotation": "matrix-radio-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "others",
+        "form_name": "ralpmh_covid_symptoms",
+        "section_header": "",
+        "field_type": "matrix-radio",
+        "field_label": "Others",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "None "
+            },
+            {
+                "code": "1",
+                "label": "Mild "
+            },
+            {
+                "code": "2",
+                "label": "Moderate "
+            },
+            {
+                "code": "3",
+                "label": "Severe"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "y",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "covid_symptom_common",
+        "matrix_ranking": "",
+        "field_annotation": "matrix-radio-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "joint_pain",
+        "form_name": "ralpmh_covid_symptoms",
+        "section_header": "Do you have any other COVID-19 Symptoms?",
+        "field_type": "matrix-radio",
+        "field_label": "Joint pain",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "None "
+            },
+            {
+                "code": "1",
+                "label": "Mild "
+            },
+            {
+                "code": "2",
+                "label": "Moderate "
+            },
+            {
+                "code": "3",
+                "label": "Severe"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[others] = 1 or [others] = 2  or [others] = 3",
```

Comment:
Also I think we should add single quotes here for the logic values since the code is parsed as a string.

Reply:
ok updated now